content/nap-dos/deployment-guide/learn-about-deployment.md

```diff
@@ -52,6 +53,7 @@ See the NGINX Plus full list of prerequisites for more details. NGINX App Protec
 - gRPC, HTTP/2 and WebSocket protection require active monitoring of the protected service. The directive `app_protect_dos_monitor` is mandatory for the attack to be detected.
 - TLS fingerprint feature is not used in CentOS 7.4 and RHEL 7 / UBI 7 due to the old OpenSSL version. The required OpenSSL version is 1.1.1 or higher.
 - Monitor directive `app_protect_dos_monitor` with proxy_protocol parameter can not be configured on Ubuntu 18.04. As a result, gRPC and HTTP/2 DoS protection for proxy_protocol configuration is not supported.
+- Regularly update the Operating System (OS) to avoid known OS vulnerabilities which may impact the service.
```

Comment:
Review note: this PR is to a dedicated branch (`dos-release-4.6`), and this line does _not_ show up in the frontdoor build. 

However, I verify this shows up "fine" in my local build of the branch, raanan_dos_release-4.6.